src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/JsonMarkerGenesControllerWIT.java

```diff
@@ -1,10 +1,6 @@
 package uk.ac.ebi.atlas.experimentpage.markergenes;
 
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.*;
```

Comment:
its better to import only those methods which we need rather than importing all of them. Its sometime IntelliJ tries to be smart and replace it with *.